Sprint-1/exercises/paths.js

```diff
@@ -8,11 +8,25 @@
 // └──────┴──────────────┴──────┴─────┘
 
 // (All spaces in the "" line should be ignored. They are purely for formatting.)
-
+console.log()
 const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
 const lastSlashIndex = filePath.lastIndexOf("/");
 const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
+console.log()
+
+
 // Create a variable to store the dir part of the filePath variable
+const dir=filePath.slice(lastSlashIndex-43,44)
```

Comment:
Can you rewrite the code so that it will work for any file path (and not just the sample value)?